### DISCUSSION TITLE
[ FF7 ] Game crash trying to enter Icicle Village

### DISCUSSION THREAD

**Discussion by Murphy1976** [julianxhokaxhiu/FFNx#520] [Q&A] [answered]
Running 7th Heaven 2.2.3.522

Tried running 2.7.1 - could not get game to load, launcher kept crashing, so resorted to downgrade 7th Heaven just to run the game with mods.

Game will launch properly.
Save file loads properly. 

Player is just outside Icicle Village. when I attempt to enter village game will crash:

Oops! Something very bad happened.
Wrote emergency save to save/crash.ff7 dir
Please provide a copy of those files along with FFNx.LOG to this github forum.

[Crash_and_FFNx.zip](https://github.com/julianxhokaxhiu/FFNx/files/10333011/Crash_and_FFNx.zip)

Surprisingly enough, 7th Heaven 2.0.3.406 runs just fine - no crashes or issues.

**Comment by julianxhokaxhiu** [comment, marked as answer]
You're using a very old 7th installation and this is not the right place to ask for support. Please use the Discord channels for it.
